### PR TITLE
VIH-10993 Remove obsolete load keyvault file implementation

### PR DESCRIPTION
**Pull request by shaed-parkar**
### Jira link

VIH-10993

### Change description

* reload on config change for secret by file
* Remove deprecated VH.Core.Configuration

### CODE REVIEW

File: NotificationApi/NotificationApi/NotificationApi.csproj [hunk 1]
```diff
@@ -73,14 +72,14 @@
...
-        <Copy SourceFiles="@(ReferencePath)" DestinationFolder="$(OutDir)References" />
-        <Exec ConsoleToMSBuild="true" ContinueOnError="true" Command="$(NSwagExe_Net60) run /runtime:Net60 /variables:ProjectPath=&quot;$(MSBuildProjectFullPath)&quot;">
-            <Output TaskParameter="ExitCode" PropertyName="NSwagExitCode" />
-            <Output TaskParameter="ConsoleOutput" PropertyName="NSwagOutput" />
-        </Exec>
-        <Message Text="$(NSwagOutput)" Condition="'$(NSwagExitCode)' == '0'" Importance="low" />
-        <Error Text="$(NSwagOutput)" Condition="'$(NSwagExitCode)' != '0'" />
-        <RemoveDir Directories="$(OutDir)References" />
-    </Target>
+<!--    <Target Name="NSwag" AfterTargets="Build" Condition="'$(TF_BUILD)'!='True' And '$(Configuration)' == 'Debug' ">-->
```
Comment (oliver-scott) [inline NotificationApi/NotificationApi/NotificationApi.csproj:75]: don't we need this block? The other apps have it

File: azure-pipelines.sds.pr-release.yml [hunk 2]
```diff
@@ -92,6 +92,24 @@ stages:
               vstsFeedId: "${{ variables.nuget_org_name }}/${{ variables.nuget_feed_name }}"
               coreProjectPath: $(projectPath)
 
+  ##################################################
+  # Deploy Nuget Packages. #########################
+  - stage: Push_Nuget
```
Comment (oliver-scott) [inline azure-pipelines.sds.pr-release.yml:97]: could you try moving this between the Docker Build & Push Image step and the EF step? I believe that will allow the nuget step to "branch" off the Pre-Release Step and run at the same time as the docker step (like for bookings and video api - if you look at the pipeline diagram)

**Review by oliver-scott — COMMENTED** [review]
looks good, just a couple of comments

File: azure-pipelines.sds.pr-release.yml (hunk 2 above)
Comment (shaed-parkar) [reply azure-pipelines.sds.pr-release.yml:97]: will do, I swear this is the same as the bookings api but I'm getting a different output. Will look again

File: NotificationApi/NotificationApi/NotificationApi.csproj (hunk 1 above)
Comment (shaed-parkar) [reply NotificationApi/NotificationApi/NotificationApi.csproj:75]: oh good eye! complete forgot to revert this 😄

**Review by himesh99 — APPROVED** [review]
lgtm

**Review by SamKompfner-HMCTS — APPROVED** [review]

**Review by oliver-scott — APPROVED** [review]

**Pull request merged**